src/components/Calculator.js

```diff
@@ -2,46 +2,110 @@ import React from 'react';
 import Container from 'react-bootstrap/Container';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
+import Button from 'react-bootstrap/Button';
+import calculate from '../logic/calculator';
+
 // eslint-disable-next-line react/prefer-stateless-function
 class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: null,
+      next: null,
+      operation: null,
+    };
+    this.getAnswer = this.getAnswer.bind(this);
+  }
+
+  getAnswer = (e) => {
+    const newState = calculate(this.state, e.target.value);
+    this.setState(newState);
+    const showAnswer = document.querySelector('.answer');
+    if (newState.next !== null) {
+      showAnswer.innerHTML = newState.next;
+    } else {
+      showAnswer.innerHTML = newState.total;
+    }
```

Comment:
- [ ] I am sorry but there still is an error when the operator buttons are hit multiple times. While you take a good step, you did not fully cover the error handling. I suggest you replace this line of code in place of the highlighted/selected block of code in case you struggle to fix it.
`showAnswer.innerHTML = (newState.next && newState.next.toString())
    || (newState.total && newState.total.toString()) || '0';`

![Screenshot from 2021-08-20 05-10-57](https://user-images.githubusercontent.com/64894927/130155736-9341513d-b12d-421e-9534-68a5530f7b57.png)

Reply:
Hello, thanks for doing my review. I have fixed it using your recommendation. Kindly review my code again